e2e/cypress/support/task_commands.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/**
+* postMessageAs is a task which is wrapped as command with post-verification
+* that a message is successfully posted by the user/sender
+* @param {Object} sender - a user object who will post a message
+* @param {String} message - message in a post
+* @param {Object} channelId - where a post will be posted
+*/
+Cypress.Commands.add('postMessageAs', ({sender, message, channelId, rootId, createAt}) => {
+    const baseUrl = Cypress.config('baseUrl');
+
+    cy.task('postMessageAs', {sender, message, channelId, rootId, createAt, baseUrl}).its('status').should('be.equal', 201);
+});
```

Comment:
@thekiiingbob I reverted this code which you've introduced before and then was removed by me.  When I removed it before, for some reason, I've got some issue and I thought it's about Cypress task inside Cypress command.  I think I'm wrong as I can't repro the issue when I reverted it now.